rails-snow-queen/app/assets/javascripts/quote.js

```diff
@@ -10,40 +10,37 @@ class Quote {
   
   addListeners() {
     $("#submitQuoteModal").on("show.bs.modal", this.handleOpenModal);
-
     const quoteFormNode = document.getElementById("new_quote");
     quoteFormNode.addEventListener("ajax:error", this.handleErrors);
     quoteFormNode.addEventListener("ajax:success", this.handleSuccess);
-
-    $("#successModal").on("hide.bs.modal", () => { location.reload() }) //hacer funcion pa que se vea igual que los otros
-    //abrir el quote modal desde js y no desde html
+    $("#successModal").on("hide.bs.modal", this.handleReload)
```

Comment:
the handler isn't "handling the reload", it's handling the modal closing (it just happens to reload when that happens). Callback should be `handleSuccessModalClose` or something like that.

Reply:
done